code/modules/mob/living/carbon/human/human.dm

```diff
@@ -207,6 +209,10 @@
 					to_chat(usr,  "<span class='notice ml-1'>Detected physiological traits:</span>\n<span class='notice ml-2'>[quirkstring]</span>")
 				else
 					to_chat(usr,  "<span class='notice ml-1'>No physiological traits found.</span>")
+			if(href_list["medrecords"]) //SKYRAT EDIT ADDITION BEGIN - EXAMINE RECORDS
+				to_chat(usr, "<b>Medical Record:</b> [MED.fields["past_records"]]")
+			if(href_list["genrecords"])
+				to_chat(usr, "<b>General Record:</b> [R.fields["past_records"]]") //SKYRAT EDIT END
```

Comment:
My question to you, does this use the Medical HUD? It doesn't seem like it does and it feels like everyone would get this info displayed to them. Also, a few more tweaks.
```suggestion
			//SKYRAT EDIT ADDITION BEGIN - EXAMINE RECORDS
			if(href_list["medrecords"])
				to_chat(usr, "<b>Medical Record:</b> [med_record.fields["past_records"]]")
			if(href_list["genrecords"])
				to_chat(usr, "<b>General Record:</b> [general_record.fields["past_records"]]")
			//SKYRAT EDIT END
```

Reply:
I've already tested it in a private server, these options only appear if you have a medHUD on.

Reply:
Also, they're covered by a if(href_list["hud"] == "m")